app/javascript/dashboard/components/widgets/conversation/bubble/Text.vue

```diff
@@ -71,6 +84,37 @@ export default {
     toggleQuotedContent() {
       this.showQuotedContent = !this.showQuotedContent;
     },
+    handleClickOnContent(event) {
+      // if event target is IMG and not wrapped with A tag
+      // then open image preview
+      if (
+        event.target.tagName === 'IMG' &&
+        event.target.parentElement.tagName !== 'A'
+      ) {
+        this.openImagePreview(event.target.src);
+      }
+    },
+    openImagePreview(src) {
+      this.show = true;
+      this.attachment.data_url = src;
+      this.allAttachments = [{ ...this.attachment }];
+    },
+    onClose() {
+      this.show = false;
+      this.resetAttachments();
+    },
+    createDummyAttachment() {
```

Comment:
Why do we need to create this?

Reply:
We are using our existing `<gallery-view/>` component to render the images, so in the component, we are expecting two props, one as an object and one as an array. Thats why I added it.